include/CNZSL/WriterStates.h

```diff
@@ -31,6 +32,8 @@ CNZSL_API void nzslWriterStatesEnableOptimization(nzslWriterStates* statesPtr, n
 CNZSL_API void nzslWriterStatesEnableSanitization(nzslWriterStates* statesPtr, nzslBool enable);
 CNZSL_API void nzslWriterStatesSetDebugLevel(nzslWriterStates* statesPtr, nzslDebugLevel debugLevel);
 
+CNZSL_API void nzslWriterStatesSetFsModuleResolver(nzslWriterStates* statesPtr, const nzslFilesystemModuleResolver* resolverPtr);
```

Comment:
since this function will match an overloaded function, it would be better called `nzslWriterStatesSetModuleResolver_Filesystem` imo